runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsPlugin.java

```diff
@@ -62,4 +81,34 @@ protected void shutDown() throws Exception
 	{
 		overlayManager.remove(overlay);
 	}
+
+	@Subscribe
+	public void onGameTick(GameTick e)
+	{
+		final WorldPoint playerPos = client.getLocalPlayer().getWorldLocation();
+
+		if (lastPlayerPosition != null)
+		{
+			if (!playerPos.equals(lastPlayerPosition))
+			{
+				lastTickPlayerMoved = client.getTickCount();
+			}
+			else if (lastTickPlayerMoved + 1 == client.getTickCount())
+			{
+				lastTimePlayerStoppedMoving = System.currentTimeMillis();
```

Comment:
Let's use `System.nanoTime()` to avoid any user system clock update issues this could cause.